packages/account/src/Components/forms/idv-form.tsx

```diff
@@ -0,0 +1,309 @@
+            },
+            true
+        );
+    };
+
+    const getExampleFormat = (example_format: string) => {
+        return example_format ? localize('Example: ') + example_format : '';
+    };
+    const getDocument = (text: string) => {
+        return document_list.find(d => d.text === text);
+    };
+
+    const onKeyUp = (e: { target: HTMLInputElement }, document_name: string) => {
+        const { example_format } =
+            document_name === 'document_number' ? values.document_type : values.document_type.additional;
+        let current_input: string | null = null;
+        current_input = example_format.includes('-')
+            ? formatInput(example_format, current_input ?? e.target.value, '-')
+            : e.target.value;
+        setFieldValue(document_name, current_input, true);
+    };
+
+    return (
+        <React.Fragment>
+            <section className={classNames('idv-form', class_name)}>
+                <div className='details-form'>
+                    <div className='poi-form-on-signup__fields'>
+                        <div
+                            className={classNames('proof-of-identity__container', {
+                                'proof-of-identity__container--idv': hide_hint,
+                            })}
+                        >
+                            <div className={classNames('proof-of-identity__inner-container')}>
+                                <div className='proof-of-identity__fieldset-container'>
+                                    <fieldset className={classNames({ 'proof-of-identity__fieldset': !hide_hint })}>
+                                        <Field name='document_type'>
+                                            {({ field }: FieldProps) => (
+                                                <React.Fragment>
+                                                    <DesktopWrapper>
+                                                        <div className='document-dropdown'>
+                                                            <Autocomplete
+                                                                {...field}
+                                                                name='document_type'
+                                                                data-lpignore='true'
+                                                                error={touched.document_type && errors.document_type}
+                                                                autoComplete='off'
+                                                                type='text'
+                                                                label={localize('Choose the document type')}
+                                                                list_items={document_list}
+                                                                value={values.document_type.text}
+                                                                onBlur={(e: { target: HTMLInputElement }) => {
```

Comment:
@yauheni-deriv, Isn't the type of this (` onBlur={(e: { target: HTMLInputElement }) `)  `inferred`, do we have do redeclare it?

Reply:
It was not being inferred. TS hint was showing any